chainquery/compose/docker-compose.yml-prod-example

```diff
@@ -35,14 +36,16 @@ services:
 ## Chainquery ##
 ################
   chainquery:
-    image: lbryio/lbry-docker:linux-x86_64-production
+    build:
+      context: .
+      dockerfile: Dockerfile-linux-x86_64-production
```

Comment:
Would we prefer to build the production example rather than pull from the LBRY official repo?